src/StackExchange.Redis/StringSplits.cs

```diff
@@ -4,6 +4,9 @@ internal static class StringSplits
     {
         public static readonly char[]
             Space = { ' ' },
-            Comma = { ',' };
+            Comma = { ',' },
```

Comment:
purely observation, but in recent TFMs we *finally* have an API that resolves to `string.Split('x')` without creating an array; sadly, this doesn't help us since we multi-target :(